lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -624,6 +624,7 @@ bool CleanupSharedMemoryAddrSpaceCast(Module &M) {
                                valueMap)) {
         bConverged = false;
         bChanged = true;
+        break;
```

Comment:
@tex3d this one is for you to review

Reply:
Helena came by earlier to ask about this.  I approve of this change.